frontend/src/pages/auth/callback.tsx

```diff
@@ -1,14 +1,14 @@
 import { useRouter } from 'next/router';
 import useRedirect from '../../context/useRedirect';
 import { useEffect, useState } from 'react';
-import { useGetFetchData } from '../../context/useGetFetchData';
+import { useGetFetchDataRedirect } from '../../context/useGetFetchData';
```

Comment:
헤헤.. path 절대경로 부탁드립다ㅏㅏ

```suggestion
import { useGetFetchDataRedirect } from '@/context/useGetFetchData';
```

Reply:
랭킹 페이지 너무 좋네요 ㅎㅎ